docs/API.md

```diff
@@ -344,175 +414,184 @@ Example:
-                }
+Request body:
+```json
+{
+    "precise/wordpress-23": {
+        "Meta": {
+            "extra-info/featured": true,
+            "extra-info/revision-info": "12dfede4ee23",
+            "bad-metaname": 3235
+        }
+    },
+    "trusty/mysql-23": {
+        "Meta": {
+            "extra-info/featured": false,
         }
+    }
+}
 ```
 
-Response body (with HTTP status 500)
-
-```
-        {
-                "Message": "multiple errors (1) found":
-                "Code": "multiple errors",
-                "Info": {
-                        "precise/wordpress-23": {
-                                "Message": "multiple errors",
-                                "Code": "multiple errors",
-                                "Info": {
-                                        "bad-metaname": {
-                                                "Message": "metadata not found",
-                                                "Code": "not found"
-                                        }
-                                }
-                        }
+Response body (with HTTP status 500):
+```json
+{
+    "Message": "multiple errors (1) found",
+    "Code": "multiple errors",
+    "Info": {
+        "precise/wordpress-23": {
+            "Message": "multiple errors",
+            "Code": "multiple errors",
+            "Info": {
+                "bad-metaname": {
+                    "Message": "metadata not found",
+                    "Code": "not found"
                 }
+            }
         }
+    }
+}
 ```
 
-`GET id/meta`
+#### GET id/meta
 
-The `id/meta` path the same information as the meta path. The results are the same regardless of the actual id.
-
-Example:
+The `id/meta` path the same information as the meta path. The results are the
+same regardless of the actual id.
 
-`GET foo/meta`
+Example: `GET foo/meta`
 
-```
+```json
 [
-        "bundle-metadata",
-        "bundle-unit-count",
-        "bundle-machine-count",
-        "manifest",
-        "charm-actions",
-        "charm-config",
-        "color",
-        "archive-size",
-        "bundles-containing",
-        "extra-info",
-        "charm-related",
-        "publish-time",
-       “revision-info”
-        "tags",
+    "archive-size",
+    "archive-upload-time",
+    "bundle-machine-count",
+    "bundle-metadata",
+    "bundle-unit-count",
+    "bundles-containing",
+    "charm-actions",
+    "charm-config",
+    "charm-metadata",
+    "charm-related",
+    "extra-info",
+    "id",
+    "id-name",
+    "id-revision",
+    "id-series",
+    "id-user",
+    "manifest",
+    "revision-info",
+    "stats",
+    "tags"
 ]
 ```
 
-`GET id/meta/any?[include=meta[&include=meta...]]`
+#### GET id/meta/any
 
-The `meta/any` path returns requested metadata information on the given id. If the id is non-specific, the latest revision and preferred series for the id will be assumed.
+The `meta/any` path returns requested metadata information on the given id. If
```

Comment:
This text no longer makes much sense without the metadata include
flags (what requested metadata information?)